tests/common/plugins/conditional_mark/issue.py

```diff
@@ -56,6 +56,9 @@ def get_cred(self):
                     github_creds = creds.get(self.NAME, {})
                     self.user = github_creds.get('user', '')
                     self.api_token = github_creds.get('api_token', '')
+                else:
+                    self.user = os.environ["GIT_USER_NAME"]
+                    self.api_token = os.environ["GIT_API_TOKEN"]
```

Comment:
Please use `os.environ.get('XXX') to avoid `KeyError` exception in case the environment variables are undefined.

Reply:
done